### PR TITLE
JavaScript. Homework 3

### PR DESCRIPTION
**Pull request by VioletaScvortova**
Good Day! ☀️ 

Below is a screenshot of my homework №3. Great and very interesting tasks! There was a lot to think about, and it was very good practice for me.

JS_ Homework 3

### CODE REVIEW

**Review by sdmnc — APPROVED** [review]
The only issue I see there is only switch and ternary and no if else expressions

**Comment by VioletaScvortova** [comment]
> The only issue I see there is only switch and ternary and no if else expressions

if I understood correctly, there is no need to use if and else in switch statement and ternary oparator

**Review by IvanCeban — CHANGES_REQUESTED** [review]
Well done!
A couple of optimization notes

File: JavaScript homeworks/homework-3.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    console.log(yourFuture);
+}
+
+tellFortune("painter", "South America", "Carlos", "4"); // Output: You will be a painter in South America, and will get married to Carlos with 4 kids
+tellFortune("biologist", "Australia", "Louis", "1"); // Output: You will be a biologist in Australia, and will get married to Louis with 1 kids
+tellFortune("sculptor", "Europe", "Santiago", "3"); // Output: You will be a sculptor in Europe, and will get married to Santiago with 3 kids
+
+//Function task 2//
+
+function calculateDogAge(X) {
```
[the remark this answers is not in the document]
Comment (VioletaScvortova) [reply JavaScript homeworks/homework-3.js:145]: Done! Replaced  X with humanAge

File: JavaScript homeworks/homework-3.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+//Function task 1//
+
+function tellFortune(X, Y, Z, N) {
+
+    let childrenNumber = N;
+    let partnerName = Z;
+    let continent = Y;
+    let post = X;
+
+    let yourFuture = "You will be a " + X + " in " + Y + ", and will get married to " + Z + " with " + N + " kids"
```
[the remark this answers is not in the document]
Comment (VioletaScvortova) [reply JavaScript homeworks/homework-3.js:134]: Fixed it! Thank you, this way is much more convenient

**Comment by VioletaScvortova** [comment]
@IvanCeban Hi! I`ve made fixes based on comments and added a bonus task with the Arrow function as well
Screenshot_2

**Review by IvanCeban — APPROVED** [review]

**Pull request merged**